### PR TITLE
Update captur to 3.2

### PR DESCRIPTION
**Pull request by colindunn**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

**Comment by miccal** [comment]
The website lists:

> Download Captur 3.2 (for Mac OS X 10.10 or Later)
Download Captur 3.0 (for Mac OS X 10.10 or Later)
Download Captur 2.4 (for Mac OS X 10.10 Yosemite)
Download Captur 2.3 (for OS X Lion)
Download Captur 2.2 (for Mac OS X 10.6)

It is not clear to me that version 2.4 is for Mavericks and lower?

**Comment by colindunn** [comment]
You're right, I had misread that. After reading it again, I've changed it to yosemite as a minimum.

I don't have an older version of OSX to test the install of the other versions, so leaving them off.

**Review by miccal — DISMISSED** [review]

File: Casks/captur.rb [hunk 1]
```diff
@@ -1,11 +1,14 @@
...
 
   # dropboxusercontent.com/u/27027504 was verified as official when first introduced to the cask
-  url "https://dl.dropboxusercontent.com/u/27027504/captur-#{version}.dmg.zip"
+  url "https://dl.dropboxusercontent.com/u/27027504/captur-#{version}.zip"
   name 'Captur'
   homepage 'https://cambhlumbulunk.blogspot.com/p/captur.html'
 
-  app '64 Bit/Captur.app'
+  depends_on macos: '>= :yosemite'
+  container nested: "Captur #{version}.dmg"
```
Comment (reitermarkus) [inline Casks/captur.rb]: This is not needed. It's detecting the nested container automatically for me.

**Review by reitermarkus — CHANGES_REQUESTED** [review]

File: Casks/captur.rb (hunk 1 above)
Comment (colindunn) [inline Casks/captur.rb]: Didn't work for me when I was fixing it up, but maybe it was the messaging with the wrong app link. Updated.

**Pull request merged**